modules/osdk-run-proxy.adoc

```diff
@@ -0,0 +1,142 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/golang/osdk-golang-tutorial.adoc
+// * operators/operator_sdk/ansible/osdk-ansible-tutorial.adoc
+// * operators/operator_sdk/helm/osdk-helm-tutorial.adoc
+
+ifeval::["{context}" == "osdk-golang-tutorial"]
+:golang:
+:app-proper: Memcached
+:app: memcached
+:group: cache
+endif::[]
+ifeval::["{context}" == "osdk-ansible-tutorial"]
+:ansible:
+:app-proper: Memcached
+:app: memcached
+:group: cache
+endif::[]
+ifeval::["{context}" == "osdk-helm-tutorial"]
+:helm:
+:app-proper: Nginx
+:app: nginx
+:group: demo
+endif::[]
+
+[id="osdk-run-proxy_{context}"]
+
+= Enabling proxy support
+
+To support proxied clusters, your Operator must inspect the environment for the following standard proxy variables and pass the values to Operands:
+
+- `HTTPS_PROXY`
+- `HTTPS_PROXY`
+- `NO_PROXY`
```

Comment:
You list 3 variables here, but in the below procedures it looks like you're mostly only using HTTP_PROXY. Just wanted to check whether the other two needed to be mentioned in the procedures too or not?

Reply:
@bergerhoffer Does this clarify things? The procedure is the same, regardless of the type of proxy variable.

```
>  .Prerequisites
> * A cluster with cluster-wide egress proxy enabled.
> 
> [NOTE]
> ====
> This tutorial uses `HTTP_PROXY` as an example environment variable.
> ====
```

Reply:
@bergerhoffer I also added a link to setting up a cluster-wide egress proxy in the **Additional resources** sections.